molparse/io.py

```diff
@@ -1152,6 +1162,15 @@ def writePDB(
 
             strbuff += "REMARK " + "# Atoms:    " + str(system.num_atoms) + end
 
+        if ssbonds:
+            if not system.ssbonds:
+                mout.warningOut(f"No SS bonds information available. Please run 'system.ssbon_guesser() first'.")
+            else:
+                for i, bond in enumerate(system.ssbonds):
+                    line = constructSSBONDLine(i, bond)
```

Comment:
Rename the function constructPDBSSBONDLine because it's specific to the PDB format